README.md

```diff
@@ -153,11 +153,14 @@ Follow these [instructions](https://hatch.pypa.io/latest/install/) to install it
 ### Tests
 
 ```sh
-# Run all tests
-hatch run test
```

Comment:
I would get ride of the existing `test` script. No need to keep them around. We should all switch to use the new commands.

Reply:
Do we still want to keep `docker-hatch`, which in my opinion is still useful at least during the migration?

If so, we will need to keep the current `test script`, because for `docker-hatch -v 3.7`, it installs an older version of hatch, which doesn't support new `hatch test`.

Reply:
Yes the challenge is, hatch that works with py3.7 (hatch <= 1.10.0) won't have builtin `test`, so we will still need to run the `script test` for python 3.7, if we keep docker-hatch?